README.md

```diff
@@ -0,0 +1,16 @@
+![GitHub last commit](https://img.shields.io/github/last-commit/sdonkov/dawg)
+
+# DAWG
+
+Try to implement DAWG (Directed Acyclic Word Graph) also known as DAFSA is a library for computing Deterministic Acyclic Finite State Automata.
+DAFSA are data structures derived from tries that allow representing a set of sequences that takes the form of a directed acyclic graph with a single source vertex
+(a vertex with no incoming edges), each edge of the graph is labeled by a letter or symbol, and each vertex has at most one outgoing edge for each possible letter or symbol.
+The strings represented by the DAFSA are formed by the symbols on paths in the graph from the source vertex to any sink vertex (a vertex with no outgoing edges).
+A deterministic finite state automaton is acyclic if and only if it recognizes a finite set of strings.
+
+## GOAL
+I'm researching data structures to improve my programming skills
+
+## Installation
+
+Install Maven, running unit and performance tests from the command line.
```

Comment:
Not sure what this means. The point is to provide instructions on how to build the code and eventually contribute to it, i.e. these are instructions for developers. Place the commands that will trigger unit tests and performance tests here.